test/db.js

```diff
@@ -1,15 +1,14 @@
-'use strict'
-
 /* eslint-env mocha */
 /* eslint-disable require-atomic-updates */
+'use strict'
 
 const chai = require('chai')
 chai.use(require('chai-as-promised'))
 const expect = chai.expect
 
 const {
-  generateTests,
-  generateCode,
+  // generateTests, // TODO: find use
```

Comment:
this was copied from another project, it's basically a function that processes an array of [input, expectedOutput]

It will come in handy later when sync functions will be tested